.travis.yml

```diff
@@ -1,96 +1,60 @@
 language: cpp
+
+dist: trusty
```

Comment:
No, it will use the default one that's defined by travis. Atm. that's 12.04 – incl. completely outdated software.

Once there's a new environment, Trusty is still available (maybe it becomes the new default?)